mcp/pointing/angles.c

```diff
@@ -54,6 +80,15 @@ double wrap_to(double angle, double max)
     return angle;
 }
 
+
+/**
+ * @brief discretizing angles to integer values, this wraps
+ * the angle to the range of [0, max]
```

Comment:
a bit confusing docstring, could be interpreted as "this function takes in a doublwe and rounds it off"

Reply:
fair, trying to fix